packages/babel-helper-split-export-declaration/README.md

```diff
@@ -0,0 +1,23 @@
+# @babel/helper-split-export-declaration
+
+## API
+
+```js
+declare export default splitExportDeclaration(path: NodePath);
```

Comment:
`declare`?

Reply:
this was intended as API doc - I've followed what was already done in exmaple [here](https://github.com/babel/babel/blob/master/packages/babel-helper-bindify-decorators/README.md#api)

Reply:
Alright, I wondered if it was done somewhere else. I guess it's fine, I don't feel strongly. Just potentially distracting since not everyone knows Flowtype.